vignettes/datatable-programming.Rmd

```diff
@@ -389,6 +389,29 @@ Users will usually want to substitute the function name rather than inserting th
 
 In case of any doubts on the `env` interface functioning, set `verbose = TRUE` to inspect how expressions are resolved internally.
 
+### Use `env` argument from inside another function
+
+It was a design decision that `env` argument will follow _Standard Evaluation_ rules. As a result, it does not require any special handling when being used from inside another function. As well it does not recognize data.table's `.()` alias for a `list()`.
```

Comment:
```suggestion
It was a design decision that `env` argument will follow _Standard Evaluation_ rules. As a result, it does not require any special handling when being used from inside another function. Importantly, it does not recognize data.table's `.()` alias for a `list()`. You can add `. = list` to your `env` if you'd like.
```

Reply:
> You can add `. = list` to your `env` if you'd like.

this is wrong, it would only subtitute .() to list() inside i/j/by, and we talk here about env arg. Coming edit will make it clear.